README.md

```diff
@@ -101,6 +101,22 @@ This project welcomes contributions and suggestions.  Most contributions require
 
 When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
 
+### Quickstart with Git
+
+1. Install [Git](https://git-scm.com/downloads), [NodeJs](https://nodejs.org/en/download/) and [VS Code](https://code.visualstudio.com/Download) if you haven’t ready.
+2. Create a GitHub account if you don’t already have one.
+3. Clone [OfficeDev/Office-Addin-Scripts](https://github.com/OfficeDev/Office-Addin-Scripts)
```

Comment:
We could move the clone into the command prompt steps.

Make step 3. Go to https://github.com/OfficeDev/Office-Addin-Scripts.
Make step 4. Create a fork or the repository for yourself.

Then add a first step under "In a command prompt:" and we can improve the "git remote add" step:
        1. git clone https://github.com/OfficeDev/Office-Addin-Scripts
        2. cd Office-Addin-Scripts
        3. git remote add {username} https://github.com/{username}/Office-Addin_Scripts.git 
            (where "{username}" is your GitHub user name.) 
         4. git checkout -b branch-name
             (recommend using all lowercase with hyphens for branch names)